pandas/tests/io/parser/test_python_parser_only.py

```diff
@@ -185,8 +185,10 @@ def test_read_csv_buglet_4x_multi_index2(python_parser_only):
     tm.assert_frame_equal(result, expected)
 
 
-@pytest.mark.parametrize("add_footer", [True, False])
-def test_skipfooter_with_decimal(python_parser_only, add_footer):
+@pytest.mark.parametrize(
+    "add_footer, decimal", [(True, "#"), (False, "#"), (True, b"#"), (False, b"#")],
```

Comment:
```suggestion
    @pytest.mark.parametrize("add_footer", [True, False])
    @pytest.mark.parametrize("decimal", ["#", b"#"])
```
Can just stack the decorates instead of manually creating tuples